utils/fix_cmocka_xml.py

```diff
@@ -8,16 +8,15 @@
 import sys
 
 # Some fixes for the CMOCKA headers
-xml_header ="<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n"
+xml_header = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n"
 parent_header = "<testsuites>\n"
 parent_footer = "</testsuites>\n"
 file_extensions = "*.xml"
 path = os.path.join(os.getcwd(), "test_results") + os.path.sep
 if not os.path.isdir(path):
     print("No test_results directory found")
     sys.exit()
-files  = [path+fn for fn in os.listdir(path)
-              if any(fn.endswith(x) for x in file_extensions)]
+files = [path+fn for fn in os.listdir(path) if any(fn.endswith(x) for x in file_extensions)]
```

Comment:
Does `path+fn` not need spaces?

Reply:
doesn't seem to, I hadn't really read this code properly and looking now it's not great is it.

Reply:
I decided to not comment on the implementation, just the style :)